docs/guide/index.md

```diff
@@ -38,6 +38,8 @@ The supported template presets are:
 |  [preact](https://vite.new/preact)  |  [preact-ts](https://vite.new/preact-ts)  |
 |     [lit](https://vite.new/lit)     |     [lit-ts](https://vite.new/lit-ts)     |
 |  [svelte](https://vite.new/svelte)  |  [svelte-ts](https://vite.new/svelte-ts)  |
+|   [ember app](https://vite.new/ember-app)   |   [ember app (ts)](https://vite.new/ember-app-ts)   |
+|   [ember library](https://vite.new/ember-library)   |   [ember library (ts)](https://vite.new/ember-library-ts)   |
```

Comment:
These won't work as `vite.new` uses the template in this repository.

Reply:
Ah, that's this pr: https://github.com/vitejs/vite/pull/19564

Maybe i should resurrect it since all the blockers are dealt with now